cmd/project-api/design/types.go

```diff
@@ -0,0 +1,89 @@
+// Copyright The Linux Foundation and each contributor to LFX.
+// SPDX-License-Identifier: MIT
+
+package design
+
+import (
+	//nolint:staticcheck // ST1001: the recommended way of using the goa GSL package is with the . import
+	. "goa.design/goa/v3/dsl"
+)
+
+// Project is the DSL type for a project.
+var Project = Type("Project", func() {
+	Description("A representation of LFX Projects.")
+
+	Attribute("id", String, "Project ID -- v2 id, not related to v1 id directly", func() {
+		Example("123")
```

Comment:
Any reason not to [enforce this](https://goa.design/docs/4-concepts/1-design-language/1-data-modeling/#built-in-formats) as a UUID right now?